### PR TITLE
variance fix: calling lead, zinc gates

### PR DESCRIPTION
**Pull request by joshuareagan**
It's worth noting this is my first change involving reference counts, so please check that I didn't screw this up.

The +fire jet seems fine in testing so I'm turning it back on, unless there are any objections.

### CODE REVIEW

**Comment by joshuareagan** [comment]
Check failed :^O

What did I do?

**Comment by joshuareagan** [comment]
https://github.com/urbit/arvo/pull/902

**Comment by joemfb** [comment]
It's my fault that these CI builds are taking so long. I enabled the kernel compilation test in the %ford test suite, but I forgot that those tests would then run under garbage-collection here. I'm frankly not sure if that's possible.

File: jets/f/ut_peek.c [hunk 1]
```diff
@@ -88,11 +88,30 @@
...
               pro = _cqfu_peek(van, p_sut, way, lat);
             }
             else {
+
+              if ( way != c3__read ) {
+               u3z(lat);
+               u3z(now);
+               u3z(pec);
+               u3z(tow);
+               return u3m_error("payload-block");
```
[the remark this answers is not in the document]
Comment (joshuareagan) [reply jets/f/ut_peek.c]: I see, bailing means we don't worry about references anymore.  Good to know...

[the remark this answers is not in the document]
Comment (benjamin-tlon) [reply jets/f/ut_peek.c]: Yeah, it tears down the whole process (often in response to things being in an invalid state), so there's no reason to try to leave things in a consistent state before a bail.

[the remark this answers is not in the document]
Comment (joshuareagan) [reply jets/f/ut_peek.c]: ok, taking them out after talking with Ben -- don't want to confuse anyone

**Pull request merged**